lua/tree.lua

```diff
@@ -28,6 +29,21 @@ function M.open()
   end
 end
 
+
+local function go_to_git_item(mode)
+  local icon_state = config.get_icon_state()
+  if not icon_state.show_git_icon then return end
+
+  local icons = {}
+  for _, icon in pairs(icon_state.icons.git_icons) do
+    table.insert(icons, icon)
+  end
```

Comment:
this should be cached to improve performance. Also checkout `vim.tbl_values` function.

Reply:
Done, thanks.